src/lib.rs

```diff
@@ -427,6 +427,30 @@ impl<L: AnalysisLoader> AnalysisHost<L> {
         result
     }
 
+    pub fn borrow_info(&self, span: &Span) -> AResult<BorrowData> {
+        self.with_analysis(|a|
+            a.def_id_for_span(span)
+             .and_then(|id| a.for_each_crate(|c| {
+                trace!("Searching in crate `{}`", c.name);
+                for (_, b) in c.per_fn_borrows.iter() {
+                    // If we find a `BorrowData` where there's a matching scope, then filter out
+                    // out matching items.
+                    if b.scopes.iter().any(|a| a.ref_id == id) {
```

Comment:
I built `BorrowData` around the idea that I could quickly figure out the `fn` item wrapping a `span`. I couldn't figure out how to do that just yet. Any ideas? If it's not something that's going to be supported, I'll change `per_fn_borrows` to be a `Vec` rather than a `HashMap`.